pages/common/groff.md

```diff
@@ -12,6 +12,10 @@
 
 `groff -man -T ascii {{manpage.1}} | less`
 
+- Normalize output:
+
+`groff -man -T utf8 {{manpage.1}} | col -bpx | perl -00 -pe '' > {{page.man}}`
```

Comment:
This is beginning to become more of a different command. I don't know about this.

Reply:
When `man` output to file,
1. filter reverse line feeds: `col -bpx` comes from man-db/src/man.c:1616.
2. compress consecutive blank lines: It used a long command of `sed`. In `perl`, `-00` enable paragraph mode.